File: Assignment4/program1.c

// Write a Program to Calculate Multiplication of Factors of the Numbers

#include<stdio.h>

 int MultiplicationOfFactor(int iNo)
 {
   int iCnt = 0;
   int iFrequency = 1;
   if(iNo < 0)
   {
    iNo = -iNo;
   }

   for(iCnt = 1 ; iCnt <= (iNo/2) ; iCnt++)
   {
    if(iNo % iCnt == 0)
    {
       iFrequency = iFrequency * iCnt;
    }
   }
 return iFrequency;

 } 

int main()
{
  int iValue = 0;
  int iRet = 0;

  printf("Enter the Number");
  scanf("%d",&iValue);

  
  iRet =  MultiplicationOfFactor(iValue);

  printf("%d\n",iRet);

    return 0;
}

File: Assignment20/program1.java

// 1. Write a program to find the sum of all even numbers up to N.

package Assignment20;

import java.util.Scanner;

class program1
{
    public int Sum_Even(int iValue)
    {
        int iSum = 0,iCnt = 0;

        for(iCnt = 0; iCnt <= iValue; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                iSum = iSum + iCnt;
            }
        }
        return iSum;
    }

    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        program1 pobj = new program1();
        
        int iValue = 0,iRet = 0;

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        iRet = pobj.Sum_Even(iValue);
        System.out.println("Sum of all Even Number upto n is : "+iRet);

        sobj.close();
    }
}